polygon/bridge/bridge.go

```diff
@@ -41,7 +41,7 @@ type eventFetcher interface {
 }
 
 func Assemble(dataDir string, logger log.Logger, borConfig *borcfg.BorConfig, eventFetcher eventFetcher) *Bridge {
-	bridgeDB := polygoncommon.NewDatabase(dataDir, kv.PolygonBridgeDB, databaseTablesCfg, logger)
+	bridgeDB := polygoncommon.NewDatabase(dataDir, kv.PolygonBridgeDB, databaseTablesCfg, logger, false)
```

Comment:
there is a style guide for not leaving naked parameters (I sometimes forget this too, good to start making a habit) - https://github.com/uber-go/guide/blob/master/style.md#avoid-naked-parameters